tasks/freebsd-12.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: freebsd | install logrotate
```

Comment:
looks like you forgot to call this file from tasks/main.yml ?

@planet-winter

Reply:
Yes! It is obsolete anyway as it does the same as the freebsd.yml which is called for all freebsd versions.

I probably was on autopilot, as I usually use a similar structure to this to flexibly get custom variables. It is easy to enhance:
```
- name: gather os specific variables
  include_vars: "{{ item }}"
  with_first_found:
    - files:
        - "{{ ansible_distribution|lower }}-{{ ansible_distribution_version }}.yml"
        - "{{ ansible_distribution|lower }}-{{ ansible_distribution_release }}.yml"
        - "{{ ansible_distribution|lower }}-{{ ansible_distribution_major_version }}.yml"
        - "{{ ansible_distribution|lower }}.yml"
        - "{{ ansible_os_family|lower }}.yml"
        - defaults.yml
      paths:
        - ../vars
      skip: True
```

Reply:
just seen you are using this already.
Same could be done with tasks though.